src/sys/ptrace/linux.rs

```diff
@@ -152,6 +157,80 @@ libc_enum! {
     }
 }
 
+#[cfg(all(target_os = "linux", target_env = "gnu"))]
+#[cfg_attr(docsrs, doc(cfg(all())))]
```

Comment:
Please remove this attribute and the one in line 175 as they are [useless](https://github.com/nix-rust/nix/pull/2192) now